### ISSUE TITLE
Package() overflow crash

### ISSUE THREAD

**Issue by Moorre** [acpica/acpica#622]
Hi!
When Package() used with wrong size, then compiler crashing. ver 20200717
`Segmentation fault: 11`
`The compiler crashed with code 11.`

For example,
`Package(4) --> Package(2)`
```
            Method (_PSL, 0, Serialized)  // _PSL: Passive List
            {
                Return (Package (0x04)
                {
                    \_PR.CPU0, 
                    \_PR.CPU1
                })
            }
```

```
Mac-Pro-Steve:~ stevebond$ diff DSDT_noerr.dsl DSDT_crash.dsl 
823,824d822
<         Processor (CPU2, 0x03, 0x00000410, 0x06){}
<         Processor (CPU3, 0x04, 0x00000410, 0x06){}
1242,1244c1240
<                     \_PR.CPU1, 
<                     \_PR.CPU2, 
<                     \_PR.CPU3
---
>                     \_PR.CPU1
```
[DSDT_crash.zip](https://github.com/acpica/acpica/files/5049983/DSDT_crash.zip)

**Comment by SchmErik** [comment]
Thanks for the report. Looks like this happens only with reserved names (named objects that start with an underscore). I'll take a look. Thanks!

**Comment by SchmErik** [comment]
fix available here: https://github.com/acpica/acpica/pull/623

**Comment by Moorre** [comment]
> fix available here: #623

Now it is good, thnx!

Close this issue.

**Comment by Moorre** [comment]
Closing this issue
